src/components/LicenseFilters/TermFiltersForm.js

```diff
@@ -52,6 +52,12 @@ class TermFiltersForm extends React.Component {
     editingFilters: false,
   }
 
+  isValidNumber = (value) => {
+    return !value ? <FormattedMessage id="stripes-erm-components.errors.customPropertyInvalidNumber" />
```

Comment:
This will return an invalid for a number of `0`. I don't think that's what we want.

Reply:
Hmm... whats geting passed down in `value` is actually a string not a number. So, passing 0 will not return an ivalid? ( as !`0` === false). This validation will fail if you are entering numbers like `13.3.4` or `13.3-` as in that case `value` is passed down as `""`